lib/rubocop/cop/layout/trailing_whitespace.rb

```diff
@@ -48,7 +48,7 @@ class TrailingWhitespace < Base
 
         def on_new_investigation
           processed_source.lines.each_with_index do |line, index|
-            next unless line.end_with?(' ', "\t")
+            next unless line.end_with?(' ', "\t", "\u3000")
```

Comment:
Can you use `[[:blank:]]` instead?

```suggestion
            next unless line.match?(/[[:blank:]]\z/)
```

Reply:
That looks better. Thanks.